python/tvm/relax/pipeline.py

```diff
@@ -93,9 +94,26 @@ def get_pipeline(name: str = "zero", **kwargs) -> tvm.transform.Pass:
        The transformation pipeline.
     """
 
-    if name in PIPELINE_MAP:
-        return PIPELINE_MAP[name](**kwargs)
-    else:
+    if name not in PIPELINE_MAP:
         raise ValueError(
             f"Unknown pre-built pipeline {name}," f"candidates are {list(PIPELINE_MAP.keys())}"
         )
+    return PIPELINE_MAP[name](**kwargs)
+
+
+def register_pipeline(name: str):
```

Comment:
return annotation plz

Reply:
it's currently used as a [decorator](https://github.com/mlc-ai/mlc-llm/blob/main/python/mlc_chat/compiler/compiler_pass/pipeline.py#L14), so I suppose we don't need a type annotation by default? I'm happy to add one in a follow-up PR if you think this is the case